img/checkmark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.1" viewBox="0 0 16 16"><path d="m2.35 7.3 4 4l7.3-7.3" stroke="#000" stroke-width="2" fill="none"/></svg>
```

Comment:
Could these icons be fetched from the sharing apps or the server?

Reply:
According to [this doc](https://docs.nextcloud.com/server/latest/developer_manual/design/icons.html), it seems that we can fetch any icon from the server instead of having it in an image folder.
For having the "yourdomain" variable from the endpoint, `OC.Backbone.history.location.origin` should do the trick.

Reply:
I would prefer to use the available icons provided by the server, if there are suitable ones. The app is currently inconsistent since we also have our own star, link, .. which will presumably be fixed once the new frontend is ready.

Reply:
I found a much simpler way to fetch icons from the server.
However, since the facebook and twitter logos are not a part of the set of nextcloud icons, I had to keep them.